src/panner3d.js

```diff
@@ -5,29 +5,36 @@ define(function (require) {
   var Effect = require('effect');
 
   /**
-   * Spatializer is a class that can construct both a Spatial Panner
-   * and a Spatial Listener. The panner is based on the 
-   * Web Audio Spatial Panner Node
-   * https://www.w3.org/TR/webaudio/#the-spatializernode-interface
+   * Panner3D is based on the <a title="Web Audio Panner docs"  href=
+   * "https://developer.mozilla.org/en-US/docs/Web/API/PannerNode">
+   * Web Audio Spatial Panner Node</a>.
    * This panner is a spatial processing node that allows audio to be positioned
    * and oriented in 3D space. 
    *
-   * The Listener modifies the properties of the Audio Context Listener. 
-   * Both objects types use the same methods. The default is a spatial panner.
+   * The position is relative to an <a title="Web Audio Listener docs" href=
+   * "https://developer.mozilla.org/en-US/docs/Web/API/AudioListener">
+   * Audio Context Listener</a>, which can be accessed
+   * by <code>p5.soundOut.audiocontext.listener</code>
    *
-   * <code>p5.Panner3D</code> - Constructs a Spatial Panner<br/>
-   * <code>p5.Listener3D</code> - Constructs a Spatial Listener<br/>
    *
-   * @class Spatializer
+   * @class p5.Panner3D
    * @constructor
-   * @return {Object} p5.Panner3D Object
-   *
-   * @property {Web Audio Node} spatializer Web Audio Spatial Panning Node
-   * @property {AudioParam} spatializer.panningModel "equal power" or "HRTF"
-   * @pproperty {AudioParam} spatializer.distanceModel "linear", "inverse", or "exponential"
    */
 	p5.Panner3D = function() {
       Effect.call(this);
+
+      /**
+       *  <a title="Web Audio Panner docs"  href=
+       *  "https://developer.mozilla.org/en-US/docs/Web/API/PannerNode">
+       *  Web Audio Spatial Panner Node</a>
+       *
+       *  Properties include
+       *    -  panningModel: "equal power" or "HRTF"
```

Comment:
I think this is ok, maybe we should just add links to the Web Audio docs for these properties? [distanceModel](https://www.w3.org/TR/webaudio/#idl-def-DistanceModelType) and [panningModel](https://www.w3.org/TR/webaudio/#idl-def-PanningModelType)

Reply:
great idea, just added links